docs/tutorial/createdb.rst

```diff
@@ -0,0 +1,112 @@
+.. _ref_tutorial_createdb:
+
+Create DB
+=========
+
+First step in a brand new project is to create the DB for it:
```

Comment:
Please use "the database" instead of "the DB".

Reply:
OK, replaced it everywhere in the docs.